src/common/RAJAPerfSuite.hpp

```diff
@@ -146,7 +152,11 @@ enum KernelID {
   Algorithm_SORT,
   Algorithm_SORTPAIRS,
 
+#ifndef RUN_KOKKOS
```

Comment:
Similar to above, not sure what is going on with the guards here. If you need more info on how this works, please let me know.

Reply:
Same as above.